<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>300</width>
    <height>375</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Message Type Filter</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="..\res\vidalia.qrc" >:/images/tor_logo16.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="title" >
      <string>Tor Messages</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkTorErr" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Messages that appear when something has gone very wrong and Tor cannot proceed.</string>
            </property>
            <property name="whatsThis" >
             <string/>
            </property>
            <property name="text" >
             <string>Tor Error</string>
            </property>
            <property name="icon" >
             <iconset resource="..\res\vidalia.qrc" >:/images/msg_error.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>421</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkTorWarn" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Messages that only appear when something has gone wrong.</string>
            </property>
            <property name="text" >
             <string>Tor Warning</string>
            </property>
            <property name="icon" >
             <iconset resource="..\res\vidalia.qrc" >:/images/msg_warn.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>421</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkTorNote" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Messages that appear infrequently during normal Tor operation and are not considered errors, but you may care about.</string>
            </property>
            <property name="text" >
             <string>Tor Notice</string>
            </property>
            <property name="icon" >
             <iconset resource="..\res\vidalia.qrc" >:/images/msg_notice.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>421</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkTorInfo" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Messages that appear frequently during normal Tor operation.</string>
            </property>
            <property name="text" >
             <string>Tor Info</string>
            </property>
            <property name="icon" >
             <iconset resource="..\res\vidalia.qrc" >:/images/msg_info.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>421</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkTorDebug" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Hyper-verbose messages primarily of interest to Tor developers.</string>
            </property>
            <property name="text" >
             <string>Tor Debug</string>
            </property>
            <property name="icon" >
             <iconset resource="..\res\vidalia.qrc" >:/images/msg_debug.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>421</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="title" >
      <string>Vidalia Messages</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkVidErr" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Messages that appear when something has gone very wrong with Vidalia.</string>
            </property>
            <property name="text" >
             <string>Vidalia Error</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>391</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkVidStat" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="toolTip" >
             <string>Messages that appear freqently indicating Vidalia operation status information.</string>
            </property>
            <property name="text" >
             <string>Vidalia Status</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>391</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>221</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>chkTorErr</tabstop>
  <tabstop>chkTorWarn</tabstop>
  <tabstop>chkTorNote</tabstop>
  <tabstop>chkTorInfo</tabstop>
  <tabstop>chkTorDebug</tabstop>
  <tabstop>chkVidErr</tabstop>
  <tabstop>chkVidStat</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../res/vidalia.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>176</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
